Replicator/CookieStore.cc

```diff
@@ -27,6 +27,29 @@ using namespace litecore::websocket;
 
 namespace litecore { namespace repl {
 
+	static int kTimeOffset = -1;
+
+	static void get_time_offset()
+	{
+		if (kTimeOffset != -1) {
```

Comment:
This needs to be thread-safe since it's global state cookies can be accessed by multiple replicators.
std::call_once would be a good way to initialize this.